src/Stratis.Bitcoin.Features.SmartContracts.Wallet/SmartContractWalletExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Linq;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.SmartContracts.Wallet
```

Comment:
We could get rid of this and  Stratis.Bitcoin.Features.SmartContracts.Wallet could reference Stratis.SmartContracts.Core. 

We have to work out the .NET Standard 2.0 vs .NET Core 2.1 rules to do this though. Do we have any rules for which projects are which? It looks like the rule is that all Features should be .NET Standard 2.0, but our SmartContract feature is Core 2.1. This is because SmartContracts.Core and Executor.Reflection are also Core 2.1. Think we just bootstrapped all the way. 

@rowandh @fassadlr let's discuss!

Reply:
The FN was updated to .NET Core 2.1 in #1593. We could bring our solutions in line with this (I had a PR to do it in VSTS).